util/ssl.py

```diff
@@ -0,0 +1 @@
+SSL_FILENAMES = ["ssl.cert", "ssl.key"]
```

Comment:
After the great cleanup (#1384) `SSL_FILENAMES` will be only in `notifications/notificationmethod.py`.